exercicios/para-casa/entregas/lara-lima-mota/models/ProdutoLimpezaBiodegradavel.ts

```diff
@@ -0,0 +1,18 @@
+import { ProdutoEcologico } from "../IProdutoEcologico";
+
+export default class ProdutoLimpezaBiodegradavel implements ProdutoEcologico {
+    nome:string;
+    preco:number;
+    volume:number;
+
+    constructor(nome: string, preco: number, volume: number) {
+        this.nome = nome;
+        this.preco = preco;
+        this.volume = volume;
+        console.log(`Produto ${this.nome} adicionado com sucesso!`)
+    }
+
+    exibirInformacoes(): void {
+        console.log(`Produto de limpeza Biodegradável => Nome: ${this.nome}, Preço: ${this.preco}, Volume: ${this.volume}`)
```

Comment:
aqui voce pode usar o método ${this.preco.toFixed(2)} para ter o retorno do preço com 2 casas decimais 
![Captura de tela 2024-06-18 2026412](https://github.com/reprograma/on32-ijs-s1-fundamentos-paradigmas-programacao/assets/31068578/81bea907-5959-478c-ae88-5042316e115c)